transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollChannelOption.java

```diff
@@ -18,6 +18,10 @@
 import io.netty.channel.ChannelOption;
 
 public final class EpollChannelOption<T> extends ChannelOption<T> {
+    public enum EpollDomainSocketReadMode {
```

Comment:
I wonder if this operation mode is also supported in any UNIX. Then we could probably move this some other package and rename it to `DomainSocketReadMode`.